dockerbuild/idsync-cron

```diff
@@ -2,5 +2,5 @@
 # See https://crontab.guru/ for help with cron pattern.
 
 # m h dom mon dow user command
-2 0 * * * root /data/run-full-batch.sh > /proc/1/fd/1 2>&1
-*/5 * * * * root /data/run-incremental-batch.sh > /proc/1/fd/1 2>&1
+*/5 * * * * root /data/run-full-batch.sh > /proc/1/fd/1 2>&1
+#*/5 * * * * root /data/run-incremental-batch.sh > /proc/1/fd/1 2>&1
```

Comment:
If this was only used for local development, I'd be fine with it. This file is used (by default) even for production usage, though, if it isn't overwritten.

Are you actually letting the cron process trigger things during local development, or are you running the sync manually?

Reply:
I didn't realize this was not just for development. Since no production instances use the incremental sync, this is probably still a worthwhile change. That would make it a breaking change, though, so maybe not worth doing.

I've only done minimal development on this in years, but I've used both manual and scheduled sync.